js/renovation/widget.tsx

```diff
@@ -269,6 +268,18 @@ export default class Widget {
         return () => active.off(this.widgetRef, { selector, namespace });
     }
 
+    @Effect()
+    clickEffect() {
+        const namespace = 'UIFeedback';
+
+        dxClick.on(this.widgetRef,
+            () => this.onClick!(this.clickArgs),
+            { namespace }
+        );
+
+        return () => dxClick.off(this.widgetRef, { namespace });
+    }
```

Comment:
- should we make the subscription without onClick property?
- should we combine `namespace` with `thus.name`?

Reply:
1) Widget (and Button) still work even if we won't pass `onClick`. Do you mean that?
2) You mean that we should make  `namespace = `UIFeedback${this.name}`?
Also, the 'this.name' at now is always `Widget` (even if it is `Button`). I think we should fix it.

Reply:
About 2
![image](https://user-images.githubusercontent.com/15093892/73350888-ef94e500-429e-11ea-9671-7ce71aef1fae.png)
Original button code